engines/pink/gui.cpp

```diff
@@ -146,10 +146,9 @@ struct SaveStateDescriptorTimeComparator {
 	}
 };
 
-static SaveStateList listSaves(bool isPeril) {
```

Comment:
Instead of removing static, you can pass the targetName as an argument in place of the boolean.

Reply:
Eh, the implicit `this` argument is just as expensive to pass, if not cheaper.

Reply:
You still need to access the _targetName member... So that's equivalent.
And by passing the target name by reference it's like passing a pointer.
This minimizes the changeset.

Reply:
Now I need two members from the class. It's now less params to pass.